go-junit-report.go

```diff
@@ -37,7 +37,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	if setExitCode && report.Failures() > 0 {
+	if setExitCode && report.Failed() {
```

Comment:
Revert this. If we introduce a failing test for build errors we can keep using `Failures()`.